src/index.js

```diff
@@ -357,10 +361,14 @@ router
   })
   .get("/likes/:feed", async ctx => {
     const { feed } = ctx.params;
-
     const likes = async ({ feed }) => {
-      const messages = await post.likes({ feed });
-      return listView({ messages });
+      const pendingMessages = post.likes({ feed });
+      const pendingName = about.name(feed);
+      return likesView({
+        messages: await pendingMessages,
+        feed,
+        name: await pendingName
+      });
```

Comment:
Don't need to use Promise.all - this is another way to run the promises in parallel